### PR TITLE
[Rasheed Mudasiru]-Sage-build a full stack mobile dapp with react native and webjs on celo

### PR DESCRIPTION
**Pull request by Taiwrash**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-16-2023-build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo/build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+
+- TailwindCSS
+
+- Web3.JS
+
+##	Initial Setup
+
+Having a basic knowledge of the above stacks will get us up to speed on getting the project set up as quickly as possible. At first, we kick start the journey from our development environment, and the tool we will be using is Expo -  “Expo tools and services empower you to create incredible apps using Expo CLI and the Expo SDK, the Expo Go app, our cloud build and submission services, and Expo Snack”. 
+
+For this project, we Expo-CLI. To install Expo-CLI, run the command
```
Comment (developerkunal) [inline blog/03-16-2023-build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo/build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo.md:50]: add a space line here

File: blog/03-16-2023-build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo/build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo.md [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+		</Send>
+		)}
+		messagesContainerStyle="flex-1 bg-gray-100"
+	/>
+	);
+}
+```
+With the help of the ```handleSend``` function on the send properties of the GiftedChat Component, we can send messages. And the user attribute helps the user handle the sender's credentials, which we can access using ```user._id```, ````user.name```, and finally, a placeholder for better accessibility.
+
+Here, the renderComposer prop was used to style the chat input box and the ```renderSend``` prop to style the send button. We also use the messagesContainerStyle prop to set the background colour of the chat messages.
```
Comment (developerkunal) [inline blog/03-16-2023-build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo/build-a-full-stack-mobile-dapp-with-react-native-and-webjs-on-celo.md:306]:
```suggestion
Here, the renderComposer prop was used to style the chat input box and the `renderSend` prop to style the send button. We also use the messagesContainerStyle prop to set the background colour of the chat messages.
```

**Comment by Taiwrash** [comment]
@developerkunal You can check now, all resolved! Thanks

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Review by Ishanpathak1 — APPROVED** [review]
Approved

**Pull request merged**